### DISCUSSION TITLE
Muzei featured art licence

### DISCUSSION THREAD

**Discussion by pihentagy** [romannurik/muzei#771] [General]
Hi!

Just to doublecheck: can I use the feed at [muzeiapi.appspot.com](https://muzeiapi.appspot.com/featured) to use it in another free app called [Variety](https://github.com/varietywalls/variety)?

**Comment by romannurik** [comment]
Hi there, unfortunately no, the API is not intended to be consumed by other apps. It's not protected (e.g. domain or SHA1 key restricted), but unfortunately we can't support other products using it.

**Reply by pihentagy** [reply]
Understood, thanks for the reply.
